releasenotes/notes/1.3/oxidize-transpiler-passes-16f2f1c57612e74a.yaml

```diff
@@ -0,0 +1,53 @@
+---
+features_transpiler:
+  - |
```

Comment:
Thanks for consolidating these into a single note. I think you just need to unify the language here. We should just list out the passes that were ported and say they're faster in general. We don't need any of the specific details for each pass.